### ISSUE TITLE
TableFeatureClassOperations - sample is missing table and dockpanes are blank

### ISSUE THREAD

**Issue by RileyQuartic** [Esri/arcgis-pro-sdk-community-samples#58]
The TableFeatureClassOperations example under Editing revolves around a table called Population by County. I can't find this in the GDBs from the most recent 3.3 sample resources data or the oldest from 2020. I'm unable to replicate the example workflow because of this. Also, when opening the tools from the ribbon, both dockpanes (common table operations and common feature class operations) show up completely blank. I've tried selecting records from a new table or from one of the feature classes in the GDBs but it doesn't make a difference. Unsure if this is related to the missing table.

**Comment by WorldlyCartos** [comment]
I can't find this table either. Thinking I'll just need to create it myself.

**Comment by Wolf-K** [comment]
You are correct the table was missing from the sample data.   We uploaded the updated c:\Data\Admin folder in this patch:  CommunitySampleData-02-03-2025-Patch.zip
You can download the patched data from the 'Assets' under the latest release: 
https://github.com/Esri/arcgis-pro-sdk-community-samples/releases/tag/3.4.0
